src/System Application/App/Cryptography Management/src/CryptographyManagement.Codeunit.al

```diff
@@ -330,6 +330,18 @@ codeunit 1266 "Cryptography Management"
         CryptographyManagementImpl.SignData(DataInStream, SignatureKey, HashAlgorithm, SignatureOutStream);
     end;
 
+    /// <summary>
+    /// Computes the hash value of the specified string and signs it.
+    /// </summary>
+    /// <param name="InputString">Input string for signing.</param>
+    /// <param name="XmlString">The private key to use in the hash algorithm.</param>
+    /// <param name="HashAlgorithm">The available hash algorithms are MD5, SHA1, SHA256, SHA384, and SHA512.</param>
+    /// <param name="SignatureOutStream">The stream to write the signature for the specified string.</param>
+    procedure SignDataRSAPSS(InputString: Text; XmlString: Text; HashAlgorithm: Enum "Hash Algorithm"; SignatureOutStream: OutStream)
```

Comment:
Should this be generic instead of just for RSAPSS?
The signature padding allows for PKCS as well. Making that a required parameter.